src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.BeforeCommon.targets

```diff
@@ -92,6 +92,24 @@ Copyright (c) .NET Foundation. All rights reserved.
     <Reference Include="System.Xml"/>
   </ItemGroup>
 
+  <!-- Set the default versions of the NETStandard.Library or Microsoft.NETCore.App packages to reference.
+       The implicit package references themselves are defined in Microsoft.NET.Sdk.props, so that they can be overridden
+       in the project file. -->
+  <PropertyGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETStandard' And '$(NetStandardDefaultPackageVersion)' == ''">
+    <NetStandardDefaultPackageVersion>1.6.0</NetStandardDefaultPackageVersion>
```

Comment:
FWIW, 1.6.1 is essential for Xamarin as my users have hit issues installing 1.6.0 into Xamarin Studio projects. 1.6.1 fixes this, and should IMO be the default for all libraries to prevent user pain.